packages/compass-web/src/index.tsx

```diff
@@ -152,6 +163,40 @@ const CompassWeb = ({
   const [connectionError, setConnectionError] = useState<any | null>(null);
   const dataService = useRef<DataService>();
 
+  const loggerProvider = useRef({
+    createLogger: (component = 'SANDBOX-LOGGER'): LoggerAndTelemetry => {
+      const logger = (name: 'debug' | 'info' | 'warn' | 'error' | 'fatal') => {
+        return (...args: any[]) => {
+          logging.push({ name, component, args });
+        };
+      };
+
+      const track = (event: string, properties: any) => {
+        tracking.push({ event, properties });
+      };
+
+      const debug = createDebug(`mongodb-compass:${component.toLowerCase()}`);
```

Comment:
I made debug() just pass through to the console as usual. Now it does the same thing as in usual electron compass. We never had that output in files anyway.